examples/2-query/src/GetAll.re

```diff
@@ -0,0 +1,89 @@
+/* Opens our styles module */
+open GetAllStyles;
+
+type state = {
+  listOfPokemons: Js.Array.t(string),
+  textInput: string,
+  filteredList: Js.Array.t(string),
+  selectedPokemon: Js.Nullable.t(string),
+};
+
+type action =
+  | ChangeInput(string)
+  | SelectPokemon(string);
+
+/* Filters our pokemon list by our input */
+let filteredPokemonList = (~pokemonList: Js.Array.t(string), ~input: string) =>
+  Js.Array.filter(
+    pokemon => Js.String.includes(input, Js.String.toLowerCase(pokemon)),
```

Comment:
Do you need to call `.toLowerCase` on `input` as well to ensure case insensitivity?

Reply:
yep good point!